core/control-plane/control-plane-aggregate-services/src/main/java/org/eclipse/edc/connector/service/query/QueryValidator.java

```diff
@@ -83,7 +85,10 @@ protected Result<Void> isValid(String path) {
 
             // cannot query on extensible (=Map) types
             if (type == Map.class) {
-                return Result.failure("Querying Map types is not yet supported");
+                Pattern pattern = Pattern.compile("^[0-9A-Za-z.':/]*$");
+                Matcher matcher = pattern.matcher(path);
+                return matcher.find() ? Result.success() :
+                        Result.failure("Querying Map types is not yet supported");
```

Comment:
The error message should be consistent with the validation

Reply:
Can this PR  merged and taken up in https://github.com/eclipse-edc/Connector/issues/3731. Map type can not be supported if it is not in specified format and hence reused same.